openedx_caliper_tracking/processor.py

```diff
@@ -9,6 +9,8 @@
 from openedx_caliper_tracking.base_transformer import base_transformer, page_view_transformer
 from openedx_caliper_tracking.caliper_config import EVENT_MAPPING
 from openedx_caliper_tracking.loggers import get_caliper_logger
+from openedx_caliper_tracking.tasks import deliver_caliper_event_to_kafka
```

Comment:
Move this import into some flag condition to avoid server errors if kafka settings are not set

Reply:
done.